### PR TITLE
Add mock-vler files

### PR DESCRIPTION
**Pull request by jjennings-va**
https://vasdvp.atlassian.net/browse/SSS-36

added mock-vler to deployer

promotatron PR: https://github.com/department-of-veterans-affairs/health-apis-promotatron/pull/15

### CODE REVIEW

File: products/mock-vler.conf [hunk 1]
```diff
@@ -0,0 +1,7 @@
+DU_ARTIFACT=health-apis-mock-vler
+DU_VERSION=1.0.0
+DU_NAMESPACE=mvler
```
Comment (jblefkove-va) [inline products/mock-vler.conf]: Let use `mock-vler` for the namespace

File: products/mock-vler.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  name: $DU_NAMESPACE
+  labels:
+    deployment-id: $K8S_DEPLOYMENT_ID
+    deployment-unit: $PRODUCT
+    deployment-unit-artifact: $DU_ARTIFACT
+    deployment-unit-version: $DU_VERSION
+    deployment-date: $BUILD_DATE
+    deployment-s3-bucket: $DU_AWS_BUCKET_NAME
+    deployment-s3-folder: $DU_S3_FOLDER
+    deployment-app-version: $MEE_VERSION
```
Comment (jblefkove-va) [inline products/mock-vler.yaml]: MOCK_VLER_VERSION

**Comment by bschofield-va** [comment]
I can merge this after @mramirez-va or @sbair-va give it the ol' 👍

**Comment by jblefkove-va** [comment]
Troubleshooting this branch on Jenkins... don't merge quite yet

**Comment by jblefkove-va** [comment]
:shipit:

**Pull request merged**